workflow/rules/common.smk

```diff
@@ -280,4 +291,25 @@ def all_input(wildcards):
             )
         )
 
-    return wanted_input
+    if config["experiment"]["3-prime-rna-seq"]["activate"]:
+        wanted_input.extend(
+                expand("results/QC/{unit.sample}-{unit.unit}.aligned.txt", unit=units.itertuples())
+            )
+
+        wanted_input.extend(
+            expand("results/kallisto_3prime/{unit.sample}-{unit.unit}", unit=units.itertuples())
+        )
+        wanted_input.extend(
+            expand("results/aligned_reads/{unit.sample}-{unit.unit}.read_names.txt",unit=units.itertuples())
+        )
+        wanted_input.extend(
+            expand("results/kallisto_cds/{unit.sample}-{unit.unit}",unit=units.itertuples())
+        )
+        wanted_input.extend(
+            expand("results/canonical_reads/{unit.sample}-{unit.unit}.fastq",unit=units.itertuples())
+        )
+        if config["experiment"]["3-prime-rna-seq"]["plot-qc"] != "all":
+            wanted_input.extend(
+                expand("results/QC/{ind_transcripts}.QC_plot.html",ind_transcripts=config["experiment"]["3-prime-rna-seq"]["plot-qc"])
+            )
```

Comment:
Please check which of these really need to be global targets. *aligned.txt probably not, the kallisto output folders neither, nor the fastqs or the read_names.txt

Reply:
Noted. Changed to required ones.